src/interactive-window/interactiveWindow.ts

```diff
@@ -454,7 +454,9 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
         let saved = true;
         const file = getFilePath(fileUri);
         // Make sure the file is saved before debugging
-        const doc = this.documentManager.textDocuments.find((d) => this.fs.areLocalPathsSame(d.fileName, file));
+        const doc = this.documentManager.textDocuments.find((d) =>
+            this.fs.arePathsSame(Uri.parse(d.fileName), Uri.parse(file))
```

Comment:
d.fileName should be replaced with d.uri
file should be replaced with fileUri

Reply:
On it!